### PR TITLE
Feat/add email configuration options

### PR DESCRIPTION
**Pull request by lyz-code**
Added `emailPort` and `emailTLS` configurations in `values.yaml`

To be reviewed after #17

### CODE REVIEW

File: charts/weblate/templates/deployment.yaml [hunk 1]
```diff
@@ -78,6 +78,10 @@ spec:
                   key: admin-password
             - name: WEBLATE_EMAIL_HOST
               value: "{{ .Values.emailHost }}"
+            - name: WEBLATE_EMAIL_PORT
+              value: "{{ .Values.emailPort }}"
+            - name: WEBLATE_EMAIL_USE_TLS
+              value: "{{ .Values.emailTLS }}"
```
Comment (nijel) [inline charts/weblate/templates/deployment.yaml:84]: When doing this, can you please add WEBLATE_EMAIL_USE_SSL as well?

**Comment by tarioch** [comment]
One thing to note, the chart allows to configure any environment variable by using the extraConfig section. The thought was to not having to update the chart all the time and just use this section.

File: charts/weblate/templates/deployment.yaml (hunk 1 above)
Comment (lyz-code) [reply charts/weblate/templates/deployment.yaml:84]: Added in ebd5032c27ca01efe4403937158cb20001ac7095

**Comment by lyz-code** [comment]
I realized that after doing the PR :(

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**